docs/cloud/managing-airbyte-cloud.md

```diff
@@ -23,6 +23,15 @@ To add a user to your workspace:
     The user will have access to only the workspace you invited them to. Also note that they will be added as a workspace admin by default.
     :::
 
+### Remove users from your workspace​
+
+To remove a user from your workspace:
+
+1. On the Airbyte Cloud dashboard, click **Settings** in the navigation bar. 
```

Comment:
@Amruta-Ranade should I add spaces between these list items for now until we restructure the doc?